### PR TITLE
Head of Personnel drip

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Resprites the head of personnel
image

Ports the HoP default uniform/skirt and HoP coat (with minor tweaks) from tg https://github.com/tgstation/tgstation/pull/67286

Removes the human resources dress outfit. This outfit's sprite will be recycled into the dress for the Chief Engineer.

## Why It's Good For The Game
Better looking outfits

## Images of changes
Before:
image

After:
image

Now featuring: Dog hats
image

## Testing
Above images
Tested all species sprites for errors

## Changelog
:cl:
add: Ports the HoP uniform/skirt and coat from tg, the original uniform is now formal
del: Removes the HoP human resources dress for now
imageadd: Resprites the HoP dress, HoP whimsy, HoP mantle, HoP beret and HoP cap
/:cl:

### CODE REVIEW

**Comment by Oyu07** [comment]
image
image

Love the rest, just miss the vibrancy of the cape

Can you show how it looks with jumpsuit + cape?

**Comment by Christasmurf** [comment]
> image image
> 
> Love the rest, just miss the vibrancy of the cape
> 
> Can you show how it looks with jumpsuit + cape?

image

File: code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -5,42 +5,48 @@
...
-	desc = "A blue jacket and red tie, with matching red cuffs! Snazzy. Wearing this makes you feel more important than your job title does."
+/obj/item/clothing/under/rank/civilian/hop/formal
+	name = "head of personnel's formal uniform"
+	desc = "A stylish choice for a formal occasion."
+	icon_state = "hop_formal"
+	item_color = "hop_formal"
+
+/obj/item/clothing/under/rank/civilian/hop/whimsy
+	desc = "A blue sweater and red tie, with matching red cuffs! Snazzy. Wearing this makes you feel more important than your job title does."
 	name = "head of personnel's suit"
```
Comment (Henri215) [inline code/modules/clothing/under/jobs/civilian.dm:38]:
```suggestion
	name = "head of personnel's suit"
	desc = "A blue sweater and red tie, with matching red cuffs! Snazzy. Wearing this makes you feel more important than your job title does."
```

**Review by Henri215 — COMMENTED** [review]
Found no issues with the sprites

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by DGamerL — APPROVED** [review]

**Comment by Danchi299** [comment]
I always thought TG HOP sprites are quite dim
but hey, more suits are more suits

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**